lib/phoenix/pubsub.ex

```diff
@@ -112,14 +117,14 @@ defmodule Phoenix.PubSub do
   @doc """
   Subscribes the caller to the PubSub adapter's topic.
 
-    * `server` - The Pid registered name of the server
+    * `server` - The registered name of the server
```

Comment:
But pids aren't supported.  All of the non-deprecated function heads for `subscribe` have a `when is_atom(server)` guard clause.  Only atoms are supported.